recipes/tbb/all/conanfile.py

```diff
@@ -1,7 +1,8 @@
 import os
 from conans import ConanFile, tools
-from conans.model.version import Version
+from conans.client.tools import intel
```

Comment:
Is not Version an internal import?

Like msvc the intel compiler needs to load an environment, and compilervars is the equivalent to vcvars. The conan CMake and vcproj build helpers automatically handle this, but for other build systems this environment still needs to be loaded. The only real viable alternative to copy hundreds of lines of code from the conan internals to find the appropriate script to run.

Reply:
Version is no longer an internal import since Conan 1.15. All externals are listed here: https://github.com/conan-io/conan/blob/develop/conans/tools.py

I strongly recommend opening an issue asking to externalizing intel instead. Anything from conans.client is considered internal and should be avoided in recipes.